internal/db/message.go

```diff
@@ -560,6 +560,15 @@ func MarkMessageAsDeleted(ctx context.Context, tx *ent.Tx, messageID imap.Intern
 	return nil
 }
 
+func MarkMessageAsDeletedAndAssignRandomRemoteID(ctx context.Context, tx *ent.Tx, messageID imap.InternalMessageID) error {
+	randomID := imap.MessageID(fmt.Sprintf("DELETED-%v", imap.NewInternalMessageID().String()))
```

Comment:
```suggestion
	randomID := imap.MessageID(fmt.Sprintf("DELETED-%v", imap.NewInternalMessageID()))
```

should auto-call String() since it satisfies Stringer interface

Reply:
Done